### PR TITLE
[+] Add product link in merchant email

### PR DESCRIPTION
**Pull request by Eolia**

### CODE REVIEW

File: mailalerts.php [hunk 1]
```diff
@@ -289,15 +289,16 @@ public function hookActionValidateOrder($params)
 					$customization_text = preg_replace('/---<br \/>$/', '', $customization_text);
 			}
 
+        		$link = new Link();
```
Comment (jeromenadaud) [inline mailalerts.php]: Hi Eolia,

Thank you for your contribution, why do not use $context->link ?

Best regards.

Comment (Eolia) [inline mailalerts.php]: Yes, oblivion fixed.

Thanks

**Comment by jeromenadaud** [comment]
Hi,

thank you for your contribution.

Best regards.

**Comment by Eolia** [comment]
Why this merged commit has been deleted in last versions ? (on some parts...)

**Pull request merged**